lib/LedgerSMB/Template/DB.pm

```diff
@@ -141,8 +141,11 @@ sub get_from_file {
     my $content = '';
     open my $fh, '<', $path
         or die "Failed to open template file $path : $!";
-    $content .= $_ while <$fh>;
-    close $fh;
+    {
+        local $/ = undef;
+        $content =  <$fh>;
+    }
+    close $fh or die "Cannot close file $path";
```

Comment:
@ehuelsmann can you please critique this entire block of change.
It doesn't look right to me, feels like we may end up loosing bytes.
I am probably wrong though